libs/perun/components/src/lib/back-button/back-button.component.html

```diff
@@ -8,5 +8,5 @@
   class="mb-2 mt-2"
   (click)="goBack()"
   matTooltip="{{'Back' | translate}}">
-  <mat-icon class="rotate">arrow_right_alt</mat-icon>
+  <mat-icon class=" ml-auto rotate">arrow_right_alt</mat-icon>
```

Comment:
Again now it looks the same without this class I think.
```suggestion
  <mat-icon class="rotate">arrow_right_alt</mat-icon>
```